portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,77 +33,73 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. TODO: modify this file to handle tstls data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private int maxComments = 3;
+  private int maxTstls = 3;
```

Comment:
I recommend using testimonials for variable name here and everywhere instead of Tstls. I know it is more verbose, but I think it helps readability here. I had to pause and think about what tstls meant and figure out by reading code below.

Reply:
okay yeah i debated it and thought tstls might be better since shorter, but i'll go ahead and change